pkgmgmtutils/update_formulas.go

```diff
@@ -42,6 +42,10 @@ type FormulaOptions struct {
 	LinuxShaRegex   string
 	WindowsShaRegex string
 
+	// if true, open a PR even if this version is something like "x.y.z-beta1"
+	// per https://docs.brew.sh/Acceptable-Formulae#stable-versions, this is not allowed for homebrew-core
+	PublishPreRelease bool
```

Comment:
pre-release has a different meaning for git releases. e.g., our beta releases are not marked as pre-releases. a better name for this would be `PublishUnstableRelease` or something of the like, thoughts?